TACC/TACC-Docs
# chore(core-styles): use but overwrite new accent color

## Overview / Changes

- The `core-styles.base.css` now defaults to blue accent color (instead of purple).
- The `core-styles.docs.css` now changes to purple accent color (instead of using default).

## Related

- requires https://github.com/TACC/Core-Styles/pull/149
- accompanied by https://github.com/TACC/Core-CMS/pull/617
- accompanied by https://github.com/TACC/tup-ui/pull/203

## Testing

1. Verify link color is _still_ purple.
2. Verify link color is purple because global variable from `core-styles.docs.css` overwrites `core-styles.base.css`.

## UI

| `core-styles.docs.css`| `core-styles.base.css` |
| - | - |
| ![core-styles-docs-css png](https://user-images.githubusercontent.com/62723358/229173011-1fd478a0-d3aa-4fe7-957e-b4d841be06a3.png) | ![core-styles-base-css](https://user-images.githubusercontent.com/62723358/229173017-1a5c85da-1a34-4033-b7cf-3fefc4422430.png) |
